data/language/en-GB.txt

```diff
@@ -3697,6 +3697,8 @@ STR_6591    :Staff member is currently fixing a ride and can’t be fired.
 STR_6592    :Staff member is currently inspecting a ride and can’t be fired.
 STR_6593    :Remove park fences
 STR_6594    :Tile Inspector: Toggle wall slope
+STR_6595    :Show building costs when paused
```

Comment:
I'm not sure if that is the best description, the money effect can appear for removing things as well. I think a better description could be something like "Show money effect when paused"